f/connectors/geojson/README.md

```diff
@@ -0,0 +1,3 @@
+# `geojson_to_postgres`: Upload a GeoJSON file to the data warehouse
+
+This script imports data from a GeoJSON file into a database table. It reads a file containing spatial data, transforms the data into a structured format, and inserts it into a PostgreSQL database table. Optionally, it then delete the export file.
```

Comment:
I would highlight front-and-center some unique characteristics about the Geojson -> pg 

1. It flattens properties into their own columns
2. It's not using PostGIS


This is far from the only (or even most intuitive?) way to store geojson in the database.  For example other ways  might be:
1. just create a single JSONB column and throw the entire blob in there.
2. Install PostGIS extension, and then you can
    - represent the "geometry" more faithfully.
    - just use ogr2ogr to inject, or I wouldn't be surprised if PostGIS has other functions to help flatten json.

I'm not suggesting you switch to either of these -- just make clear what it is that we are doing.

Reply:
Also document that it looks for a uniique "id" as primary key.  And where it that, inside "properties" or outside?